components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -215,6 +217,7 @@ - (void)updateContentOffsetForPerformantScrolling:(CGFloat)contentYOffset {
       // Update the drawer's scrollView's offset to be static so the content will scroll instead.
       CGRect scrollViewBounds = self.scrollView.bounds;
       scrollViewBounds.origin.y = drawerOffset;
+      normalizedYContentOffset = drawerOffset;
```

Comment:
What does normalized mean in this context?

Reply:
I've updated the code with an elaborate explanation of what this variable entails.